### PR TITLE
Validation auroc fix

### PR DESCRIPTION
**Pull request by EthanMarx**
Rescales validation waveforms with SNRs below some threshold to the threshold

### CODE REVIEW

File: projects/sandbox/pyproject.toml [hunk 1]
```diff
@@ -100,20 +100,20 @@ outdir = "${base.basedir}/training"
...
 # optimization parameters
 batch_size = 512
 batches_per_epoch = 200
 max_epochs = 200
-max_lr = 4e-3
-lr_ramp_epochs = 75 
-weight_decay = 1e-5
+max_lr = 0.21355 
+lr_ramp_epochs = 83 
+weight_decay = 6.28608e-06
```
Comment (wbenoit26) [inline projects/sandbox/pyproject.toml:116]: These params should be set back to their defaults.

**Review by wbenoit26 — CHANGES_REQUESTED** [review]
Looks good other than the changes to the config. How many signals did we end up mapping to an SNR of 6?

File: projects/sandbox/pyproject.toml (hunk 1 above)
Comment (EthanMarx) [reply projects/sandbox/pyproject.toml:116]: Might be a good time to update to the best params from the hp search. Can you share those here?

Comment (wbenoit26) [reply projects/sandbox/pyproject.toml:116]: Good idea:
```
waveform_prob: 0.470644, 
glitch_prob: 0.420902, 
max_lr: 0.000181352, 
lr_ramp_epochs: 92, 
weight_decay: 2.81536e-06,
glitch_downweight: 0.650283
```

**Review by wbenoit26 — APPROVED** [review]

**Pull request merged**